Source/Resources/Resources.csproj

```diff
@@ -15,6 +15,7 @@
     <ProjectReference Include="../Concepts/Concepts.csproj" />
     <ProjectReference Include="../Services/Services.csproj" />
     <ProjectReference Include="../Execution/Execution.csproj" />
+    <ProjectReference Include="..\Async\Async.csproj" />
```

Comment:
I like when all the slashes are the same way :) I can fix this - and some newlines a few places.

Reply:
Done